ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1316,3 +1319,33 @@ def test_paddle_logical_xor(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# eig
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="eig",
+    dtypes_and_x=get_dtype_and_square_matrix_complex(real_and_complex_only=True),
+)
+def test_paddle_instance_eig(
+    dtypes_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+):
+    input_dtype, x = dtypes_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={"data": x},
+        method_input_dtypes=input_dtype,
+        method_all_as_kwargs_np={},
+        frontend_method_data=frontend_method_data,
+        init_flags=init_flags,
+        method_flags=method_flags,
+        frontend=frontend,
+        on_device=on_device,
+        test_values=False,
```

Comment:
You have to test it by reproducing the input from the output as the issue with some linalg functions is that ther have multiple correct values as a solution.
A sample would be as follows:
https://github.com/unifyai/ivy/blob/ca5c9aa8bb5d30fd4591fc338fe27329bcf61fcc/ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py#L177